package.json

```diff
@@ -19,10 +19,10 @@
     "csslint": "^1.0.2",
     "ethereum-remix": "https://github.com/ethereum/remix",
     "ethereumjs-abi": "https://github.com/ethereumjs/ethereumjs-abi",
-    "ethereumjs-block": "https://github.com/ethereumjs/ethereumjs-block",
-    "ethereumjs-tx": "https://github.com/ethereumjs/ethereumjs-tx",
-    "ethereumjs-util": "https://github.com/ethereumjs/ethereumjs-util",
-    "ethereumjs-vm": "https://github.com/ethereumjs/ethereumjs-vm",
+    "ethereumjs-block": "^1.6.0",
+    "ethereumjs-tx": "^1.3.3",
+    "ethereumjs-util": "^5.1.2",
```

Comment:
I checked https://github.com/ethereum/browser-solidity/issues/633 and https://github.com/ethereum/browser-solidity/issues/628 
both are fixed (even using util 5.1.2 ...)